test/test_hmc_util.py

```diff
@@ -226,7 +226,8 @@ def potential_fn(q):
     fn = (jit(find_reasonable_step_size, static_argnums=(0, 1, 2))
           if jitted else find_reasonable_step_size)
     rng_key = random.PRNGKey(0)
-    step_size = fn(potential_fn, kinetic_fn, p_generator, init_step_size, m_inv, q, rng_key)
+    step_size = fn(potential_fn, kinetic_fn, p_generator, init_step_size, m_inv,
+                   (q, None, None), rng_key)
```

Comment:
These tests also need to be modified, I think.

Reply:
Thanks Neeraj! I just fix them and will merge if tests pass due to a typo (missing a comma) at setup file, which removes the dependency of numpyro on `jax`.